.agents/skills/gstack/.github/docker/Dockerfile.ci

```
@@ -32,7 +32,10 @@ RUN curl -fsSL https://bun.sh/install -o /tmp/bun-install.sh \
     && rm -f /tmp/bun-install.sh
 
 # Claude CLI
-RUN npm i -g @anthropic-ai/claude-code@2.1.150
+RUN curl -fsSL https://registry.npmjs.org/@anthropic-ai/claude-code/-/claude-code-2.1.150.tgz -o /tmp/claude-code.tgz \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)